modules/avoindata-theme/avoindata.theme

```diff
@@ -18,6 +18,23 @@ function avoindata_preprocess_region(array &$variables) {
   $variables['language'] = \Drupal::languageManager()->getCurrentLanguage()->getId();
 }
 
+function avoindata_preprocess_block(&$variables) {
+  $variables['link_map'] = [
+    'aboutopendatamenuen' => '/en/about-open-data',
+    'aboutopendatamenufi' => '/fi/tietoa-avoimesta-datasta',
+    'aboutopendatamenusv' => '/sv/api-principer',
```

Comment:
Hmm, I guess that this should be something else. @Zharktas I believe we had somewhere list about these url:s, I could maybe fix it in block definition also 🤔

Reply:
Yeah it should be /information-om-oppna-data/, the list is here https://wiki.dvv.fi/display/AV/Rakenne

Reply:
Its my copy paste error in the block definition though :D

Reply:
Thanks, I missed that also in the first place